### DISCUSSION TITLE
第6节课笔记(11班)

### DISCUSSION THREAD

**Discussion by keesh0410** [InternLM/Tutorial#397] [笔记]
亲爱的实战营11班的同学们，

作为你们的助教，喵德🐱，我非常荣幸能够在这段探索大语言模型的旅程中与你们同行。🌟🐾 在InternLM的支持下，我们一起深入了解了这个前沿科技的奥秘。💡🤖

从第一次按下键盘记录我们关于大语言模型的学习和洞见，到今天，每一步都充满了发现和启示。📚✨ 我们一起探讨了大模型的潜力和挑战，见证了人工智能领域的惊人发展。🚀🌌

每一次的作业笔记不仅记录了我们对技术的理解，更是我们坚持和努力的见证。📝💪 正如爱因斯坦所说：“在中间的每一步都是必要的。” 在这个旅程中，我们学习了不仅是技术，更学会了在面对未知的勇气。🌱🌟

你们的好奇心、执着和创新精神，让这次实战营变得格外精彩。正如史蒂夫·乔布斯所言：“要打破规则，你首先要掌握它们。” 我们一起学习了大模型的规则，也一起探索了它们的无限可能。📖✏️

随着实战营的结束，我们的学习之旅并未停止。未来充满了更多关于人工智能和大语言模型的探索和挑战。让我们继续携手前进，向着未知的领域迈进。💫🌈

再次感谢大家在这次大模型实战营11班的付出和努力，以及上海AI实验室给予的宝贵支持。愿我们在未来的AI之路上，都能充满光明和希望！🤖🌟🌈

**Comment by seifer08ms** [comment]
https://zhuanlan.zhihu.com/p/678381396

**Comment by zhanghui-china** [comment]
https://zhuanlan.zhihu.com/p/678842068

**Comment by lvoooo** [comment]
https://github.com/lvoooo/internLM-learning/blob/main/6.md

**Comment by horacehht** [comment]
https://blog.csdn.net/m0_50123831/article/details/135727188

**Comment by Shengshenlan** [comment]
https://github.com/Shengshenlan/tutorial/blob/master/note/6/%E7%AC%AC6%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0.md

**Comment by morning-start** [comment]
https://flowus.cn/morningstar/share/36e5213f-5de5-4ede-a9aa-a48fb708147a
【FlowUs 息流】大模型评测

**Comment by BarryYin** [comment]
https://github.com/BarryYin/internLM/blob/main/%E7%AC%AC%E5%85%AD%E8%8A%82.md

**Comment by Kongsea** [comment]
笔记：https://blog.csdn.net/deeepsea/article/details/135815463

**Comment by JustLike1** [comment]
https://blog.csdn.net/qq_45607888/article/details/135774659

**Comment by GMYelims** [comment]
https://blog.csdn.net/weixin_42112760/article/details/135787337?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135787337%22%2C%22source%22%3A%22weixin_42112760%22%7D

**Comment by wwewwt** [comment]
https://github.com/wwewwt/-InternLM/wiki/6%E3%80%81opencompase

**Comment by GMYelims** [comment]
https://blog.csdn.net/weixin_42112760/article/details/135831439?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135831439%22%2C%22source%22%3A%22weixin_42112760%22%7D
进阶

**Comment by GMYelims** [comment]
https://blog.csdn.net/weixin_42112760/article/details/135832389?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135832389%22%2C%22source%22%3A%22weixin_42112760%22%7D

**Comment by wuyingxue1230** [comment]
https://zhuanlan.zhihu.com/p/679858728

**Comment by wuyingxue1230** [comment]
https://zhuanlan.zhihu.com/p/679858728

**Comment by codefd** [comment]
https://blog.csdn.net/googteo/article/details/135869712?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135869712%22%2C%22source%22%3A%22googteo%22%7D
